Makefile

```diff
@@ -218,14 +218,20 @@ include $(MAKEFILE_SPLIT)
 
 COURSE_ASM_FILES := $(wildcard courses/*/*/packed.s)
 
-C_FILES := $(foreach dir,$(SRC_DIRS),$(wildcard $(dir)/*.c))
+# These are files that need to be encoded into EUC-JP in order for the ROM to match
+# We filter them out from the regular C_FILES since we don't need nor want the
+# UTF-8 versions getting compiled
+EUC_JP_FILES := src/credits.c src/code_80005FD0.c
+C_FILES := $(filter-out $(EUC_JP_FILES),$(foreach dir,$(SRC_DIRS),$(wildcard $(dir)/*.c)))
 S_FILES := $(foreach dir,$(ASM_DIRS),$(wildcard $(dir)/*.s)) $(COURSE_ASM_FILES)
```

Comment:
This seems unnecessarily complicated.

What's wrong with compiling the utf8 verisons? They aren't linked. I don't see any issue with compiling them if it keeps the makefile a tad cleaner.

Thoughts?

Please update the readme to include iconv as a dependency and perhaps note that it generally comes packaged with gnu/linux.

Reply:
Nothing really wrong with it, just wasted work (not that they take much time to compile of course). I'll remove the filter out though, no big deal.

As for the readme, I've done some googling and it seems like `inconv` is just part of GNU C library. Presumably any system that has stuff like GCC installed is going to naturally have `iconv`.